.gitignore

```diff
@@ -0,0 +1 @@
+build_artefacts
```

Comment:
Though I agree that `artifacts` might be better. This was named to be consistent with [`conda-smithy`]( https://github.com/conda-forge/conda-smithy/blob/f97e45181b9e1f8131ad5d9f323322d1694cef78/conda_smithy/templates/run_docker_build.tmpl#L22 ) and thus existing feedstocks.

Still we could discuss changing it. We just would want @pelson on-board.